docs/mindsdb_sql/sql/create/trigger.mdx

```diff
@@ -106,7 +106,9 @@ ON mindsdb_slack.messages
         SELECT 'slack-bot-channel-id' AS channel_id, answer AS text
         FROM chatbot_model m
         JOIN TABLE_DELTA s
-        WHERE  s.user != 'U07J30KPAUF'
+        WHERE s.user != 'slack-bot-id' # this is to prevent the bot from replying to its own messages
+        AND s.channel_id = 'slack-bot-channel-id'
+        AND s.created_at > LAST
```

Comment:
I think this condition won't work: `s.created_at > LAST` 
- TABLE_DELTA already has only delta rows. 
- and it is not a real table, it is like a dataframe, it shouldn't work with LAST, as  I know

Reply:
@ea-rus 
Without the LAST condition, the bot replies to all messages from the channel every time. When I added this LAST condition, then it started working as expected.
Please see this video - https://www.loom.com/share/19acff65c4484e03a6a086371283c249

Reply:
@MinuraPunchihewa 
Let's update it as follows please:
```
JOIN TABLE_DELTA s
WHERE s.user != 'slack-bot-id' # this is to prevent the bot from replying to its own messages
AND s.channel_id = 'slack-bot-channel-id'
```

Reply:
Done!

Reply:
Thanks @MinuraPunchihewa - ready to merge.

Thanks @ea-rus for catching it - all works fine with TABLE_DELTA.